services/src/main/java/org/jboss/windup/web/services/service/MigrationProjectService.java

```diff
@@ -112,7 +112,7 @@ public void deleteOldProvisionalProjects(int olderThanMinutes)
         if (olderThanMinutes > 0)
             query += " AND pr.lastModified < DATEADD('MINUTE', -"+olderThanMinutes+", CURRENT_TIMESTAMP) "; // JPA had issues with a param here.
         List<MigrationProject> provisional = entityManager.createQuery(query, MigrationProject.class).getResultList();
-        LOG.info("Deleted " + provisional.size() + " provisional projects older than "+olderThanMinutes+" minutes.");
+        LOG.info("Deleting " + provisional.size() + " provisional projects older than "+olderThanMinutes+" minutes.");
         provisional.forEach(this::deleteProject);
```

Comment:
I tend to add before, because if the further execution fails, it's better to know what it was doing rather than not. And you are rarely interested in what happened after it happened if it went fine.